src/parser.rs

```diff
@@ -299,6 +299,22 @@ impl<'a> Parser<'a> {
         parse_single(tlv.data())
     }
 
+    /// This is an alias for `read_element::<Explicit<T, tag, 1>>` for use when
+    /// MSRV is < 1.51.
```

Comment:
Now that MSRV is >1.51, do we need these at all?

Reply:
No we actually do not. These are used int the derive crate though. But I also exchanged them with a more generic part, that take a TagClass as parameter.